packages/react-app/src/helpers/Web3ModalSetup.js

```diff
@@ -33,7 +33,7 @@ const web3ModalSetup = () =>
             10: "https://mainnet.optimism.io", // xDai
             100: "https://rpc.gnosischain.com", // xDai
             137: "https://polygon-rpc.com",
-            31337: "http://localhost:8545",
+            31337: `http://${process.env.HARDHAT_HOST || "localhost"}:8545`,
```

Comment:
@ferric-sol I am going to add an export to `constants.js` with the name of `HARDHAT_HOST` so if you could update to use the new import like:

```javascript
import { ALCHEMY_KEY, INFURA_ID, HARDHAT_HOST } from "../constants";
```
on line 6 in `Web3ModalSetup.js`

and then update this line to reflect the new import
```javascript
31337: `http://${HARDHAT_HOST || "localhost"}:8545`,
```

🚨 see PR #955

Reply:
cool, done!

Reply:
guess i could remove || localhost as well, right? @codenamejason